package.json

```diff
@@ -171,13 +170,13 @@
     "lint-staged": "14.0.1",
     "madge": "6.1.0",
     "msw": "1.3.0",
-    "nx": "16.7.4",
+    "nx": "16.8.1",
     "patch-package": "8.0.0",
     "prettier": "3.0.3",
-    "rollup": "3.20.2",
+    "rollup": "3.29.0",
```

Comment:
did you doublecheck module federation with this version? I had issues when I tried to update rollup in the past

Reply:
Yes. I saw no issues with the building and running the SDK with plugins.
By any chance, do you remember the issue you faced previously?

Reply:
I was failing to retrieve any plugin chunks